hook-worker/src/worker.rs

```diff
@@ -140,11 +141,16 @@ impl<'p> WebhookWorker<'p> {
     }
 
     /// Run this worker to continuously process any jobs that become available.
-    pub async fn run(&self, transactional: bool) -> Result<(), WorkerError> {
+    pub async fn run(
+        &self,
+        transactional: bool,
+        liveness: HealthHandle,
+    ) -> Result<(), WorkerError> {
         let semaphore = Arc::new(sync::Semaphore::new(self.max_concurrent_jobs));
 
         if transactional {
             loop {
+                liveness.report_healthy().await;
```

Comment:
These might need to move into `wait_for_job`. Those loop internally and don't yield a job until one is found.

I suppose it's unlikely to take that long in prod, but it could happen, and it'll probably happen in `dev`.